runtime/near-vm-logic/src/logic.rs

```diff
@@ -864,7 +867,7 @@ impl<'a> VMLogic<'a> {
                 .ok_or(HostError::IntegerOverflow)?;
         }
         use_gas = use_gas.checked_add(burn_gas).ok_or(HostError::IntegerOverflow)?;
-        self.gas_counter.deduct_gas(burn_gas, use_gas)
+        self.gas_counter.pay_action_accumulated(burn_gas, use_gas, Actions::new_receipt)
```

Comment:
Hm, I might be wrong, but doesn't this has all the unused gas as well?
E.g. if we attach 100Tgas and use 20Tgas, this will be 80% usage because it will record 80 Tgas?

Reply:
it only records burnt gas in the profiler.